etna/ext/completions/extconf.rb

```diff
@@ -0,0 +1,20 @@
+# Rubygem doesn't provide a great hook for executing completion or shared data on installing a gem by default,
+# but it does provide an extension installation hook in the form of extconf.rb.
+# We are 'hacking' this behavior to do a user installation of completion files on the system
+require 'mkmf'
+
+# Create a dummy extension file. Without this RubyGems would abort the
+# installation process
+FileUtils.touch(File.join(Dir.pwd, 'wat.' + RbConfig::CONFIG['DLEXT']))
+
+# Drop the completion file in the user's home directory so they can wire this based on their system.
+Dir.chdir(File.expand_path('..', __FILE__)) do
+  puts `cp ../../etna.completion ~/etna.completion`
+  puts "etna.completion has been copied to your home directory.  Source it from your bashrc file to add completions for the etna command."
```

Comment:
Oh, I see this now. I don't get this message when I install the gem -- should I?

```
$ gem install etna
Building native extensions. This could take a while...
Successfully installed etna-0.1.16
Parsing documentation for etna-0.1.16
Installing ri documentation for etna-0.1.16
Done installing documentation for etna after 0 seconds
1 gem installed
```

Reply:
Huh.  Ok, it may be surpressing stdout because it's a jerk.  Yeah, I'm not super excited about this behavior, ruby gems kind of cock blocks us from doing a bunch of things.

Reply:
Oh well...that'll be an educate-the-user thing I guess. At least you only do it once!